client/src/lib/components/+game-ui/widgets/my-lands/widget-my-lands.svelte

```diff
@@ -7,7 +7,7 @@
   import { useDojo } from '$lib/contexts/dojo';
   import { gameSounds } from '$lib/sfx';
   import { moveCameraTo } from '$lib/stores/camera.store';
-  import { claimAllOfToken } from '$lib/stores/claim.store.svelte';
+  import { claimALL, claimAllOfToken } from '$lib/stores/claim.store.svelte';
```

Comment:
`claimAll` instead of `claimALL`

Casing will cause issues later (talking from experience)
```suggestion
  import { claimAll, claimAllOfToken } from '$lib/stores/claim.store.svelte';
```